examples/platform/silabs/LEDWidget.h

```diff
@@ -31,11 +31,18 @@ class LEDWidget
     void Blink(uint32_t changeRateMS);
     void Blink(uint32_t onTimeMS, uint32_t offTimeMS);
     void Animate();
+    uint8_t GetLED();
+    bool GetLEDStatus(uint8_t led);
+    void SetLevel(uint8_t aLevel);
+    uint8_t GetLevel();
 
 private:
+    uint8_t mLed;
+
+protected:
```

Comment:
why is protected needed here now before they were all under private

Reply:
While I was adding setLed functions for RGB Led modified to protected. But that is not required now.